gpytorch/constraints/constraints.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+import math
+from torch.distributions.constraints import _Interval
+from torch.nn.functional import softplus, sigmoid
+from ..utils.transforms import _get_inv_param_transform
+from torch.nn import Module
+
+
```

Comment:
Are we assuming every constraint has a transform and inv_transform method? If so, we should probably have some abstract base class here that enforces that. Or even better, just implements nullop transforms.